nswag/Client.Class.ProcessResponse.liquid

```diff
@@ -48,7 +48,7 @@ throw new {{ ExceptionClass }}("{{ response.ExceptionDescription }}", status_, o
 throw responseObject_;
 {%                 endif -%}
 {%             else -%}
-throw new {{ ExceptionClass }}<{{ response.Type }}>("{{ response.ExceptionDescription }}", status_, objectResponse_.Text, headers_, objectResponse_.Object, null);
+throw {{ ExceptionClass }}.Create(status_, headers_, objectResponse_.Object, null);
```

Comment:
This line handles the case when throwing a documented status code when there is a response body returned. In most cases, an `ApiException` with a `ProblemDetails` with full information will be thrown. The exception to this case is `ImportDocument` that does not have a `ProblemDetails`, but instead a `CreateEntryResult`. See this commit to view what changes this line made to the generated code.
https://github.com/Laserfiche/lf-repository-api-client-dotnet/pull/61/commits/c5caa4d7f3b25b2adc08d741ac7b88fe188ff786